hoot-core/src/main/cpp/hoot/core/schema/TagComparator.cpp

```diff
@@ -524,8 +526,13 @@ void TagComparator::mergeNames(Tags& t1, Tags& t2, Tags& result)
     }
     else if (result.contains(it2.key()))
     {
-      QStringList sl = Tags::split(it2.value());
-      altNames.insert(sl.begin(), sl.end());
+      const Qt::CaseSensitivity caseSensitivity =
+        _caseSensitive ? Qt::CaseSensitive : Qt::CaseInsensitive;
+      if (result[it2.key()].compare(it2.value(), caseSensitivity) != 0)
+      {
```

Comment:
Nevermind. This is a non-issue. I got confused b/c github was showing "void TagComparator::compareEnumeratedTags(..." and then the change. This change is inside a name-only function so I think it is legit.